Steps4Impact/Supporting Files/Steps4Impact.entitlements

```diff
@@ -7,6 +7,8 @@
 	<key>com.apple.developer.healthkit</key>
 	<true/>
 	<key>com.apple.developer.healthkit.access</key>
-	<array/>
+	<array>
+		<string>health-records</string>
```

Comment:
Pretty sure that we do not need this.  This is for clinical health records which we do not need, we only need the activity access.